src/main/java/org/geysermc/discordbot/listeners/ErrorAnalyzer.java

```diff
@@ -72,66 +64,115 @@ public void onMessageReceived(MessageReceivedEvent event) {
-                embedBuilder.addField(exceptionTitle, foundFix.get().getValue(), false);
-                continue;
+        if (exceptions.size() != 0) {
+            // Get the github trees for fetching the file paths
+            String branch = "master";
+            Matcher branchMatcher = BRANCH_PATTERN.matcher(pasteBody);
+            if (branchMatcher.find()) {
+                branch = branchMatcher.group(1);
             }
+            GithubFileFinder fileFinder = new GithubFileFinder(branch);
+
+            for (StackException exception : exceptions) {
+                // Can't have more than 12 embed fields as its longer than the 6k characters on average
+                if ((embedBuilder.getFields().size() - limitCount) + limitSize >= 12) {
+                    break;
+                }
+
+                String exceptionTitle = exception.getException() + ": " + exception.getDescription();
+
+                // If there is a fix for the exception, add it.
+                if (addFixIfPresent(exceptionTitle, embedBuilder)) {
+                    continue;
+                }
 
-            for (StackLine line : exception.getLines()) {
-                if (line.getStackPackage() != null && line.getStackPackage().startsWith("org.geysermc") && !line.getStackPackage().contains("shaded")) {
-                    // Get the file url
-                    String lineUrl = fileFinder.getFileUrl(line.getSource(), Integer.parseInt(line.getLine()));
+                // If no fix exists then add some info about the error
+                for (StackLine line : exception.getLines()) {
+                    if (line.getStackPackage() != null && line.getStackPackage().startsWith("org.geysermc") && !line.getStackPackage().contains("shaded")) {
+                        // Get the file url
+                        String lineUrl = fileFinder.getFileUrl(line.getSource(), Integer.parseInt(line.getLine()));
 
-                    // Build the description
-                    String exceptionDesc = "Unknown fix!\nClass: `" + line.getJavaClass() + "`\nMethod: `" + line.getMethod() + "`\nLine: `" + line.getLine() + "`\nLink: " + (lineUrl != "" ? "[" + line.getSource() + "#L" + line.getLine() + "](" + lineUrl + ")" : "Unknown");
+                        // Build the description
+                        String exceptionDesc = "Unknown fix!\nClass: `" + line.getJavaClass() + "`\nMethod: `" + line.getMethod() + "`\nLine: `" + line.getLine() + "`\nLink: " + (!lineUrl.isEmpty() ? "[" + line.getSource() + "#L" + line.getLine() + "](" + lineUrl + ")" : "Unknown");
 
-                    // Make sure we dont already have that field
-                    if (embedBuilder.getFields().stream().noneMatch(field -> exceptionTitle.equals(field.getName()) && exceptionDesc.equals(field.getValue()))) {
                         embedBuilder.addField(exceptionTitle, exceptionDesc, false);
-                    }
```

Comment:
These changes mean that there is no check if the exceptionDesc already exists as a field's value, but I think that the check for the title will be enough (unless there are a bunch of different error messages with the same cause, I guess)

Reply:
Oh, 83a44a0 also removes the "We don't currently have automated responses for the detected errors!" message, which IIRC was sent if there was any stacktrace error from anywhere

Reply:
Yes we need to add that back